src/mpc-hc/FavoriteOrganizeDlg.cpp

```diff
@@ -449,3 +459,29 @@ void CFavoriteOrganizeDlg::OnLvnGetInfoTipList(NMHDR* pNMHDR, LRESULT* pResult)
 
     *pResult = 0;
 }
+
+
+void CFavoriteOrganizeDlg::CopyToClipboard()
+{
+    CAtlList<CString>* pSL = &m_sl[m_tab.GetCurSel()];

```

Comment:
I haven't checked the code. But does GetCurSel() always return >= 0 ?
Just making sure this doesn't crash in case nothing is selected.

Reply:
Well, this function is called only when 'Files' tab is selected, there is a condition in `OnKeyPressed` function:
```cpp
                if(m_tab.GetCurSel() == 0) {   // Files
                    CopyToClipboard();
                }
```
I can add a condition to check valid selection, or at least an assert, but that would need to be done in `SetupList()` too, because that would fail first.